tst/EnergyPlus/unit/PoweredInductionUnits.unit.cc

```diff
@@ -0,0 +1,264 @@
+		"    0;                       !- Hourly Value",
+		"Schedule:Constant,",
+		"    AlwaysOn,               !- Name",
+		"    ,                        !- Schedule Type Limits Name",
+		"    1;                       !- Hourly Value",
+
+	} ) ;
+
+	ASSERT_FALSE( process_idf( idf_objects ) );
+
+	DataGlobals::NumOfTimeStepInHour = 1; // must initialize this to get schedules initialized
+	DataGlobals::MinutesPerTimeStep = 60; // must initialize this to get schedules initialized
+	ScheduleManager::ProcessScheduleInput(); // read schedules
+	ScheduleManager::ScheduleInputProcessed = true;
+	DataEnvironment::Month = 1;
+	DataEnvironment::DayOfMonth = 21;
+	DataGlobals::HourOfDay = 1;
+	DataGlobals::TimeStep = 1;
+	DataEnvironment::DSTIndicator = 0;
+	DataEnvironment::DayOfWeek = 2;
+	DataEnvironment::HolidayIndex = 0;
+	DataEnvironment::DayOfYear_Schedule = General::JulianDay(DataEnvironment::Month, DataEnvironment::DayOfMonth, 1);
+	DataEnvironment::StdRhoAir = Psychrometrics::PsyRhoAirFnPbTdbW( 101325.0, 20.0, 0.0 );
+	ScheduleManager::UpdateScheduleValues();
+
+	bool ErrorsFound = false;
+	HeatBalanceManager::GetZoneData(ErrorsFound);
+	ASSERT_FALSE(ErrorsFound);
+	DataZoneEquipment::GetZoneEquipmentData1();
+	ZoneAirLoopEquipmentManager::GetZoneAirLoopEquipment();
+	Fans::GetFanInput();
+	Fans::GetFanInputFlag = false;
+	PoweredInductionUnits::GetPIUs();
+	EXPECT_TRUE( compare_err_stream( "" ) );
+	DataHeatBalFanSys::TempControlType.allocate( 1 );
+	DataHeatBalFanSys::TempControlType( 1 ) = DataHVACGlobals::DualSetPointWithDeadBand;
+
+	// node number table
+	//  1   SPACE2-1 Air Node
+	//  2   SPACE2-1 Return Node
+	//  3   SPACE2-1 In Node
+	//  4   SPACE2-1 ATU Sec Node
+	//  5   SPACE2-1 ATU Fan Outlet Node
+	//  6   SPACE2-1 ATU In Node
+	//  7   SPACE2-1 Zone Coil Air In Node
+
+	DataZoneEnergyDemands::ZoneSysEnergyDemand.allocate( 1 );
+	DataZoneEnergyDemands::CurDeadBandOrSetback.allocate( 1 );
+
+	// Setup for Zone 1 VAV No Reheat
+	int ZoneNum = 1;
+	int SysNum = 1;
+	int ZoneNodeNum = 1;
+	int SecNodeNum = PoweredInductionUnits::PIU( SysNum ).SecAirInNode;
+	bool FirstHVACIteration = true;
+	Real64 SecMaxMassFlow = 0.05 * DataEnvironment::StdRhoAir; // From inputs
+
+	DataGlobals::BeginEnvrnFlag = true; // Must be true for initial pass thru InitPIU for this terminal unit
+	FirstHVACIteration = true;
+	PoweredInductionUnits::InitPIU( SysNum, FirstHVACIteration ); // Run thru init once with FirstHVACIteration set to true
+	Fans::InitFan( 1, FirstHVACIteration );
+	DataGlobals::BeginEnvrnFlag = false;
+	FirstHVACIteration = false;
+
+	// Note that the fan schedule is always off, so the PIU fan should only run if the night cycle turn on flag is true
+
+	// First test - Heating load, TurnZoneFansOn is false - expecting no flow
+	DataZoneEnergyDemands::ZoneSysEnergyDemand(1).RemainingOutputRequired = 2000.0; // Heating load - expect min flow rate
+	DataZoneEnergyDemands::CurDeadBandOrSetback(1) = false;
```

Comment:
You don't adjust this value throughout all the unit tests here.  Was there a reason you copied this line into each block below?  I just want to make sure you weren't intending on also adding another set with ` = true;`.